tests/Fixer/Phpdoc/PhpdocNoEmptyReturnFixerTest.php

```diff
@@ -123,6 +123,19 @@ public function testOtherDoNothing()
      * @return int|null
      */
 
+EOF;
+
+        $this->doTest($expected);
+    }
+
+    public function testYetAnotherDoNothing()
+    {
+        $expected = <<<'EOF'
+<?php
+    /**
+     * @return null[]|string[]
```

Comment:
could you add a test to make sure the change is case-insensitive, i.e. `NULL[]` and `VOID[]` get fixed as well?

Reply:
Nice catch, they didn't because `Annotation::getTypes` isn't normalized. Fix incoming.